solr/solrj/src/java/org/apache/solr/common/params/GroupParams.java

```diff
@@ -66,5 +66,10 @@
   public static final String GROUP_DISTRIBUTED_SECOND = GROUP + ".distributed.second";
 
   public static final String GROUP_DISTRIBUTED_TOPGROUPS_PREFIX = GROUP + ".topgroups.";
+
+  /** activates optimization in case only one document per group.
+   * Setting this to true is only compatible with group.limit = 1
+   */
+  public static final String GROUP_SKIP_DISTRIBUTED_SECOND = GROUP + ".skip.second.step";
```

Comment:
How about using this `GROUP_SKIP_DISTRIBUTED_SECOND` constant more widely throughout the code e.g. when logging and in tests? https://github.com/cpoerschke/lucene-solr/commit/6bdf8723564a584dd060f32925beb3bcba68c4ad commit sketches how it might be done. This could help code readers locate code related to the optimisation.

Reply:
Done in c5553ed2ee5